s/scala/scala-2.11.12-ubi8.sh

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------
+#
+# Package	: scala
+# Version	: 2.11.12
+# Source repo	: https://github.com/scala/scala.git
+# Tested on	: rhel_8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Punith Basavarajaiah<Punith.Basavarajaiah@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+#!/bin/bash
+
+# install tools and dependent packages
+yum install -y git wget curl unzip nano vim make gcc gcc-c++
+
+# install Java packages
+yum install -y java java-devel
+
+#Set java home
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.302.b08-0.el8_4.ppc64le
+export PATH=$PATH:$JAVA_HOME/bin
+
+#install sbt package
+rm -f /etc/yum.repos.d/bintray-rpm.repo
```

Comment:
do we need this line? as far as I know the bintray-rpm repo is not supported on P anyway

Reply:
Yes correct , this line is not required